src/simulator/simulator.h

```diff
@@ -60,31 +60,37 @@ class Simulator {
   config_reader::ConfigReader reader_;
   config_reader::ConfigReader init_config_reader_;
 
-  Pose2Df vel_;
-  Pose2Df cur_loc_;
-
   std::vector<std::unique_ptr<EntityBase>> objects;
 
-  ros::Subscriber initSubscriber;
+  struct RobotPubSub {
+    Pose2Df vel;
+    Pose2Df cur_loc;
+
+    ros::Subscriber initSubscriber;
```

Comment:
might be good idea to make naming consistant (either make all variable name end with _ or not)

Reply:
Converted the one publisher that was non-standard to standard.